src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -371,26 +372,44 @@ def post(self, request: Request, organization) -> Response:
                 status=400,
             )
 
+        if is_active_staff(request):
```

Comment:
can we write a test for this? i'm thinking we might have to pass in an extra param if we're hitting this endpoint from `_admin`. otherwise if you have an active staff cookie at all, if you try to add someone to an org you're going to add them without inviting

also the staff cookie is still feature flagged so this will not work at the moment

Reply:
If we plan to add an extra param, do we hide the param in `sentry`? I feel like I should be making a new endpoint because it does feel like the endpoint does two separate things

Reply:
If the staff cookie is feature flagged, should I use the ff, to switch between checking for `superuser_has_permission` and `is_active_staff`?

Reply:
what about a param like `skip_invite`? so users who are directly POSTing could use it too?

i think an API in getsentry would not be bad. imo, ideally admin endpoints would be isolated and not overlap with sentry endpoints. if you want to pull out differing functionality i could see a case for it

you should check for the flag, yes

Reply:
Hmm, I think your idea of `skip_invite` seems good

Reply:
I agree with cathy. The endpoint there could also be much simpler than this because a lot of params here are not needed for admin.